<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDataInput</class>
 <widget class="QWidget" name="FormDataInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="m_lblEngineName">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="locale">
      <locale language="Russian" country="Ukraine"/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="m_tabWidget">
     <property name="locale">
      <locale language="Russian" country="Ukraine"/>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <widget class="QWidget" name="tab_engines">
      <attribute name="icon">
       <iconset resource="../images_resources.qrc">
        <normaloff>:/images/turbina.png</normaloff>:/images/turbina.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Engines</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="m_gboxEngineData">
         <property name="locale">
          <locale language="Russian" country="Ukraine"/>
         </property>
         <property name="title">
          <string>Engine data</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2">
           <widget class="QLineEdit" name="m_leIdData">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>65</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="m_lblCommentsTitle">
            <property name="locale">
             <locale language="Russian" country="Ukraine"/>
            </property>
            <property name="text">
             <string>Comments:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="5">
           <widget class="SelectEditPB" name="m_pbEditStartDevices">
            <property name="locale">
             <locale language="Russian" country="Ukraine"/>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="m_lblChamberTitle">
            <property name="locale">
             <locale language="Russian" country="Ukraine"/>
            </property>
            <property name="text">
             <string>Combustion chamber:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="3">
           <widget class="QLineEdit" name="m_leChamberData">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <spacer name="verticalSpacer">
            <property name="locale">
             <locale language="Russian" country="Ukraine"/>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>108</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QLabel" name="m_lblStartDevicesQntyTitle">
            <property name="locale">
             <locale language="Russian" country="Ukraine"/>
            </property>
            <property name="text">
             <string>Start devices quantity:</string>
            </property>
            <property name="buddy">
             <cstring>m_sboxStartDevicesQntyData</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer">
            <property name="locale">
             <locale language="Russian" country="Ukraine"/>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>265</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="5">
           <widget class="SelectEditPB" name="m_pbEditFuels">
            <property name="locale">
             <locale language="Russian" country="Ukraine"/>
            </property>
           </widget>
          </item>
          <item row="6" column="2" rowspan="2" colspan="4">
           <widget class="QPlainTextEdit" name="m_pteComments"/>
          </item>
          <item row="3" column="5">
           <widget class="SelectEditPB" name="m_pbEditChambers">
            <property name="locale">
             <locale language="Russian" country="Ukraine"/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="m_lblStartDeviceTitle">
            <property name="locale">
             <locale language="Russian" country="Ukraine"/>
            </property>
            <property name="text">
             <string>Start device:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QSpinBox" name="m_sboxStartDevicesQntyData">
            <property name="maximumSize">
             <size>
              <width>65</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="locale">
             <locale language="Russian" country="Ukraine"/>
            </property>
            <property name="maximum">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="m_lblFuelTitle">
            <property name="locale">
             <locale language="Russian" country="Ukraine"/>
            </property>
            <property name="text">
             <string>Fuel:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_lblFullNameTitle">
            <property name="locale">
             <locale language="Russian" country="Ukraine"/>
            </property>
            <property name="text">
             <string>Full name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="SelectEditPB" name="m_pbEditFullName">
            <property name="locale">
             <locale language="Russian" country="Ukraine"/>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="3">
           <widget class="QLineEdit" name="m_leFullNameData">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="m_lblIdTitle">
            <property name="locale">
             <locale language="Russian" country="Ukraine"/>
            </property>
            <property name="text">
             <string>Id:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="MChTypeLabel" name="m_lblModelChangeType">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
           </widget>
          </item>
          <item row="4" column="2" colspan="3">
           <widget class="QLineEdit" name="m_leStartDeviceData">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2" colspan="3">
           <widget class="QLineEdit" name="m_leFuel">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_bypasses">
      <attribute name="icon">
       <iconset resource="../images_resources.qrc">
        <normaloff>:/images/rotated_trash.png</normaloff>:/images/rotated_trash.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Bypasses</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="m_tvDBTContent">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_algorithm">
      <attribute name="icon">
       <iconset resource="../images_resources.qrc">
        <normaloff>:/images/algorithm.png</normaloff>:/images/algorithm.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Algorithm</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_settings">
      <attribute name="icon">
       <iconset resource="../images_resources.qrc">
        <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Settings</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_documents">
      <attribute name="icon">
       <iconset resource="../images_resources.qrc">
        <normaloff>:/images/documents.png</normaloff>:/images/documents.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Documents</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_graphs">
      <attribute name="icon">
       <iconset resource="../images_resources.qrc">
        <normaloff>:/images/graphs.png</normaloff>:/images/graphs.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Graphs</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="locale">
        <locale language="Russian" country="Ukraine"/>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_tbRecordFirst">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Go to the first engine</string>
       </property>
       <property name="locale">
        <locale language="Russian" country="Ukraine"/>
       </property>
       <property name="icon">
        <iconset resource="../images_resources.qrc">
         <normaloff>:/images/record_first.png</normaloff>:/images/record_first.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_tbRecordPrev">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Go to the previous engine</string>
       </property>
       <property name="locale">
        <locale language="Russian" country="Ukraine"/>
       </property>
       <property name="icon">
        <iconset resource="../images_resources.qrc">
         <normaloff>:/images/record_back.png</normaloff>:/images/record_back.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LE_DataSend" name="m_leRecordId">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Input engine's Id</string>
       </property>
       <property name="locale">
        <locale language="Russian" country="Ukraine"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_tbRecordNext">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Go to the next engine</string>
       </property>
       <property name="locale">
        <locale language="Russian" country="Ukraine"/>
       </property>
       <property name="icon">
        <iconset resource="../images_resources.qrc">
         <normaloff>:/images/record_next.png</normaloff>:/images/record_next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_tbRecordLast">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Go to the last engine</string>
       </property>
       <property name="locale">
        <locale language="Russian" country="Ukraine"/>
       </property>
       <property name="icon">
        <iconset resource="../images_resources.qrc">
         <normaloff>:/images/record_last.png</normaloff>:/images/record_last.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MChTypeLabel</class>
   <extends>QLabel</extends>
   <header>widgets/reimplemented_widgets.h</header>
  </customwidget>
  <customwidget>
   <class>LE_DataSend</class>
   <extends>QLineEdit</extends>
   <header>widgets/reimplemented_widgets.h</header>
  </customwidget>
  <customwidget>
   <class>SelectEditPB</class>
   <extends>QPushButton</extends>
   <header>widgets/reimplemented_widgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_tabWidget</tabstop>
  <tabstop>m_leIdData</tabstop>
  <tabstop>m_leFullNameData</tabstop>
  <tabstop>m_pbEditFullName</tabstop>
  <tabstop>m_pbEditFuels</tabstop>
  <tabstop>m_leChamberData</tabstop>
  <tabstop>m_pbEditChambers</tabstop>
  <tabstop>m_leStartDeviceData</tabstop>
  <tabstop>m_pbEditStartDevices</tabstop>
  <tabstop>m_sboxStartDevicesQntyData</tabstop>
  <tabstop>m_pteComments</tabstop>
  <tabstop>m_tbRecordFirst</tabstop>
  <tabstop>m_tbRecordPrev</tabstop>
  <tabstop>m_leRecordId</tabstop>
  <tabstop>m_tbRecordNext</tabstop>
  <tabstop>m_tbRecordLast</tabstop>
 </tabstops>
 <resources>
  <include location="../images_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_leIdData</sender>
   <signal>textChanged(QString)</signal>
   <receiver>m_leRecordId</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_leRecordId</sender>
   <signal>editingFinished()</signal>
   <receiver>m_leRecordId</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
